### PR TITLE
Comment Service and Controller Refactoring 

### PR DESCRIPTION
**Pull request by deciocandido**
### Changes proposted
- Updated the delete mapping to inclued /comments on the CommentController
- Updated Comment update method on Comment Service

### CODE REVIEW

File: src/main/java/se/kth/sda/skeleton/posts/PostController.java [hunk 1]
```diff
@@ -11,12 +11,10 @@
 @RequestMapping("/posts")
 @RestController
 public class PostController {
-    PostRepository postRepository;
     PostService postService;
```
Comment (clecardona) [inline src/main/java/se/kth/sda/skeleton/posts/PostController.java:15]: What is done here ?
If we did that it means that our class `postRepository `is useless, no? but we need it
Could you explain

**Review by clecardona — CHANGES_REQUESTED** [review]
all ok for me, except the deleting of `PostRepository ` and `PostController,` 
I am not sure we can do that. Could you explain?

File: src/main/java/se/kth/sda/skeleton/posts/PostController.java (hunk 1 above)
Comment (janko101) [reply src/main/java/se/kth/sda/skeleton/posts/PostController.java:15]: Yes, we don't use it in PostController, but in PostService...So, as we extracted to PostService, it's redundant to have it here anymore

Comment (clecardona) [reply src/main/java/se/kth/sda/skeleton/posts/PostController.java:15]: alright, good for me then

**Review by clecardona — APPROVED** [review]

**Review by nh-g — APPROVED** [review]

File: src/main/java/se/kth/sda/skeleton/comments/CommentController.java [hunk 2]
```diff
@@ -67,7 +67,7 @@ public CommentController(PostRepository postRepository, CommentService commentSe
      *
      * @param commentId the id of the comment
      */
-    @DeleteMapping("/{commentId}")
+    @DeleteMapping("/comments/{commentId}")
```
Comment (janko101) [inline src/main/java/se/kth/sda/skeleton/comments/CommentController.java:70]: Format. Too spaces

**Comment by clecardona** [comment]
are you also explaining git procedure to your team ? :p

**Pull request merged**